tensorflow/compiler/mlir/lite/flatbuffer_import.cc

```diff
@@ -276,7 +276,7 @@ std::vector<T> ReadAsLittleEndian(ArrayRef<uint8_t> bytes) {
   ret.reserve(elem_count);
 
   const char* data_ptr = reinterpret_cast<const char*>(bytes.data());
-  for (int i = 0; i < elem_count; i++) {
+  for (int i = 0, end = elem_count; i < end; i++) {
```

Comment:
Would rather change type of `elem_count`

Reply:
Why not change type of `elem_count` local?

Reply:
Still think it would be better to change `elem_count` on line 275 instead of here. It's bad for readability to have `a = b` when `a` and `b` are both variables and you only do the assignment to change type

Reply:
Don't forget this one

Reply:
updated.

Reply:
We no longer need `end` now.

Reply:
Approved by mistake, we need to remove `end` as is no longer needed.

Reply:
updated.